webview/src/setup/components/ProjectUninitialized.tsx

```diff
@@ -1,28 +1,83 @@
-import { VSCodeButton } from '@vscode/webview-ui-toolkit/react'
 import React from 'react'
```

Comment:
Are we planning on adjusting our welcome tree view? Showing a link to setup instead of just showing the "Initialize Project" button for example?

Reply:
My plan was to show the same buttons in the welcome view but we can simplify and drop all that logic instead.